account_move_budget/readme/USAGE.md

```diff
@@ -0,0 +1,2 @@
+In order to set up the budget items go to *Invoicing / Accounting /
+Miscellaneous / Account Move Budgets*.
```

Comment:
The route should be Accounting / Accounting / Journals / Account Move Budgets

Reply:
Updated, Thanks